tests/FormRequestTest.php

```diff
@@ -97,4 +97,14 @@ public function testErrorResponse()
         $this->assertArrayHasKey('faults', $response);
         $this->assertArrayHasKey('note', $response);
     }
+
+    public function testWithValidator()
+    {
+        $response = $this->post('form-request', ['name' => 'young Harry', 'age' => 112,])
+            ->seeStatusCode(422)
+            ->response
+            ->getData(true);
+        $this->assertIsString($response['message']);
+        $this->assertIsArray($response['errors']);
+    }
```

Comment:
It should validate the positive cases too. check if the name exists in the errors field when validation fails. and the error should match the exact error text like `The name should start with old` or whatever is appropriate.